packages/site/pages/api/embed.js.js

```diff
@@ -20,6 +21,14 @@ function embedScript(markup, css) {
 `
 }
 
+const nftData = new NFTData({
+  provider: new ethers.providers.CloudflareProvider(),
+  historicalProvider: new ethers.providers.InfuraProvider(
+    null,
+    process.env.INFURA_PROJECT_ID
```

Comment:
does the new commit allow for using test networks by passing in a testnet project from infura?

Reply:
Yes it absolutely does, if your willing to host it yourself